app/cron.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
+"""
+    This file is what runs every 5 minutes on the heroku scheduler
```

Comment:
Am I right to assume that if you have a backlog of pending exports that you can't process during these 5 minutes the next cron run would result in duplicate exports? If yes, how would you fix it?